ui/main/src/app/modules/archives/archives.component.ts

```diff
@@ -175,9 +177,34 @@ export class ArchivesComponent implements OnDestroy, OnInit {
                 this.hasResult = page.content.length > 0;
                 page.content.forEach(card => this.loadTranslationForCardIfNeeded(card));
                 this.results = page.content;
+
+                this.updatesByCardId = [];
+                this.loadUpdatesByCardId();
             });
     }
 
+    loadUpdatesByCardId() {
+        this.results.forEach(lightCard => {
+            const filters: Map<string,  string[]> = new Map();
+            filters.set('process', [lightCard.process]);
+            filters.set('processInstanceId', [lightCard.processInstanceId]);
+            this.cardService.fetchArchivedCards(filters)
+                .pipe(takeUntil(this.unsubscribe$))
+                .subscribe((page: Page<LightCard>) => {
+                    page.content.forEach(card => this.loadTranslationForCardIfNeeded(card));
+                    this.updatesByCardId.push({mostRecent: lightCard, cardHistories: page.content.slice(1), displayHistory: false});
+                });
```

Comment:
You shall use the filter set in the first query especially dates , otherwise you end up having card update outside of the range of the filters set by user .

Reply:
As discussed this morning, we have chosen to not take into account date filters for "cardHistories".